lib/model/__init__.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, classifier, parameters={}, gs_params={}):
         self.classifier = classifier
         self.model = classifier(**parameters)
         self.parameters = parameters
-        self.gs_model = GridSearchCV(self.model, gs_params)
+        self.gs_model = GridSearchCV(self.classifier(), gs_params)
```

Comment:
This :

```
def __init__(self, classifier, parameters={}, gs_params={})
self.classifier = classifier
GridSearchCV(self.classifier(), gs_params)
```

as opposed to this :

```
def __init__(self, classifier, parameters={}, gs_params={})
self.model = classifier(**parameters)
self.gs_model = GridSearchCV(self.model, gs_params)
```
The former would mean that no `**parameters` are being passed for the vanilla model at all. Ever. I was just asking if that's the case ? If that is, then we don't need `**parameters` at all in __init__